packages/gateway/index.ts

```diff
@@ -1,54 +1,20 @@
-import { ApolloServer, gql } from 'apollo-server';
+import { ApolloServer } from 'apollo-server';
 import { ApolloGateway } from '@apollo/gateway';
-import * as console from 'console';
+import * as fs from 'fs';
+import * as path from 'path';
 
-gql`
```

Comment:
This blob here was funny. I wasn't using it anywhere but it made my IDE happy. I think this whole static analysis/IDE pleasing side of things will take some hacking, but I hope it doesn't

Reply:
perhaps a `shim.graphql` might please the gods